docs/index.asciidoc

```diff
@@ -39,7 +39,7 @@ The SNMP integration plugin includes:
 The new the `logstash-integration-snmp` plugin combines the 
```

Comment:
```suggestion
The new `logstash-integration-snmp` plugin combines the 
```

Reply:
Hi @karenzone! I've sneaky committed this small fix as well! Thanks for fixing the docs :)